application/common/service_type.py

```diff
@@ -16,15 +25,38 @@ def validate_organisation_keys(org_type_id: str, org_sub_type: str) -> None:
     Raises:
         ValidationException: Either Org Type ID or Org Sub Type is not part of the valid list
     """
-    if org_type_id in SERVICE_TYPES:
-        logger.append_keys(service_type=SERVICE_TYPES[org_type_id][SERVICE_TYPES_NAME_KEY])
-        logger.info(f"Org type id: {org_type_id} validated")
-        logger.info(f"real: {org_sub_type} expected: {SERVICE_TYPES[org_type_id][ORGANISATION_SUB_TYPES_KEY]}")
-        if org_sub_type in SERVICE_TYPES[org_type_id][ORGANISATION_SUB_TYPES_KEY]:
-            logger.info(f"Subtype type id: {org_sub_type} validated")
-        else:
-            raise ValidationException(f"Unexpected Org Sub Type ID: '{org_sub_type}'")
+    validate_organisation_type_id(org_type_id)
+    if org_sub_type in SERVICE_TYPES[org_type_id][ORGANISATION_SUB_TYPES_KEY]:
+        logger.info(f"Subtype type id: {org_sub_type} validated")
+    else:
+        raise ValidationException(f"Unexpected Org Sub Type ID: '{org_sub_type}'")
+
+
+def validate_organisation_type_id(org_type_id: str) -> None:
+    """Check if the organisation type id is valid
+
+    Args:
+        org_type_id (str): organisation type id
+    """
+    feature_flags = get_feature_flags("event-processor")
+    is_pharmacy_accepted: bool = feature_flags.evaluate(name="is_pharmacy_accepted", default=False)
```

Comment:
could we instead not store a list of accepted org type keys, then we would not need to keep updating the code when we add each new type in, simply check if the org type is in the enabled list?

Reply:
Done